tests/TestCase/Network/RequestTest.php

```diff
@@ -169,7 +169,7 @@ public function testQueryStringParsingFromInputUrl()
         $request = new Request(['url' => 'some/path?one=something&two=else']);
         $expected = ['one' => 'something', 'two' => 'else'];
         $this->assertEquals($expected, $request->query);
-        $this->assertEquals('some/path?one=something&two=else', $request->url);
+        $this->assertEquals('some/path', $request->url);
```

Comment:
I feel that the old code was 'wrong' before. The 'url' (which is what Router sees) should not include query string arguments. It should be the path segment only. The name `url` is of course not fitting and will be going away :smile: